library/src/main/scala/g8.scala

```diff
@@ -82,14 +100,21 @@ object G8 {
   }
 
   def write(in: File, out: File, parameters: Map[String, String], append: Boolean) {
-    Option(PlexusIoResourceAttributeUtils.getFileAttributes(in)) match {
-      case Some(attr) =>
-        val mode = attr.getOctalMode
-        write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
-        util.Try(ArchiveEntryUtils.chmod(out, mode, new ConsoleLogger(Logger.LEVEL_ERROR, "")))
-      case None =>
-        // PlexusIoResourceAttributes is not available for some OS'es such as windows
-        write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+    try {
+      Option(PlexusIoResourceAttributeUtils.getFileAttributes(in)) match {
+        case Some(attr) =>
+          val mode = attr.getOctalMode
+          write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+          util.Try(ArchiveEntryUtils.chmod(out, mode, new ConsoleLogger(Logger.LEVEL_ERROR, "")))
+        case None =>
+            // PlexusIoResourceAttributes is not available for some OS'es such as windows
+            write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+      }
+    }
+    catch {
+      case t : Throwable =>
+        println("Falling back to file copy for %s: %s" format(in.toString, t.getMessage))
```

Comment:
Yes. I think there are two use cases here: (1) developers creating a new template & (2) users trying to create a new project from a template. I was mostly trying to solve for the first case, as I kept running into an issue when creating a template and wasn't quite sure which file was causing the problem. But this solution might help for the second case too, by just copying the file raw to the new location and letting the user try to use the new project. 

In my particular case I had some unescaped `$`'s, and so copying the files despite ST's Exceptions would have been fine for the end-user of the template.